src/main/core/support/configuration.rs

```diff
@@ -1084,7 +1084,8 @@ pub fn parse_string_as_args(args_str: &OsStr) -> Result<Vec<OsString>, String> {
     Ok(args)
 }
 
-#[cfg(test)]
+// XXX: Won't run under miri due to foreign function calls to process_parseArgStr
+#[cfg(all(test, not(miri)))]
```

Comment:
Should we disable individual tests rather than all?

Reply:
Done